marklogic-data-hub-central/ui-custom/src/views/Detail.tsx

```diff
@@ -115,7 +115,7 @@ const Detail: React.FC<Props> = (props) => {
                     "backgroundColor": "transparent"
                   },
                   "mainStyle": {
-                    "padding-top": "8px"
+                    "paddingTop": "8px"
```

Comment:
Careful here. This PR and Ivey's PR have the same change. Shouldn't it be padding-top?

Reply:
Thanks for reviewing, @aniamy. React components take JS objects when applying CSS rules inline, so the CSS rules need to be written in camelCase. paddingTop, backgroundColor, etc...